src/main/java/de/orchound/gameoflife/model/Game.java

```diff
@@ -96,6 +117,17 @@ public void randomize() {
 		notifyBoardDataObservers();
 	}
 
+	public void clear() {
+		Arrays.fill(board.target, false);
+		board.makeCurrentStateInitial();
```

Comment:
Move these to a `clear()` method in the `Board`.
```suggestion
		board.clear();
```